gitgud/levels/__init__.py

```diff
@@ -1,23 +1,22 @@
-from collections import OrderedDict
-
-from gitgud.levels.intro import all_challenges as intro_challenges
-from gitgud.levels.merging import all_challenges as merging_challenges
+from gitgud.levels.intro import level as intro_level
+from gitgud.levels.merging import level as merging_level
 
 from gitgud.levels.util import Level
+from gitgud.levels.util import AllLevels
 
-all_levels = OrderedDict()
-
-all_levels['intro'] = Level('intro', intro_challenges)
-all_levels['merging'] = Level('merging', merging_challenges)
-
+all_levels = AllLevels(
+    [
+        intro_level,
+        merging_level
+    ]
+)
```

Comment:
Looks, good. As a style preference, let's put the ([ and ]) on the same line and change the indentation, so 
```
all_levels = AllLevels([
    intro_level,
    merging_level
])
```